src/main/java/frc/robot/commands/drive/DriveFieldOriented.java

```diff
@@ -46,9 +45,9 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    double x = this.xSupplier.get() * Constants.Drive.MAX_SPEED;
-    double y = this.ySupplier.get() * Constants.Drive.MAX_SPEED;
-    double z = this.zSupplier.get() * Constants.Drive.MAX_ANGULAR_SPEED;
+    double x = this.xSupplier.get() * drive.getMaximumSpeed();
+    double y = this.ySupplier.get() * drive.getMaximumSpeed();
+    double z = this.zSupplier.get() * drive.getMaximumAngularSpeed();
```

Comment:
It might be a good idea to cache these values, so we dont have to run a getter every cycle.

Reply:
Checked within the API, it just runs `Math.max(double instanceVar1, double instanceVar2)`, which isn't expensive enough to bother caching.